server/portal/apps/webhooks/views.py

```diff
@@ -271,9 +252,9 @@ def post(self, request, *args, **kwargs):
             logger.info("Unexpected event type")
             return HttpResponse("ERROR", status=400)
 
-        # confirm that there is a corresponding running agave job before sending notification
+        # confirm that there is a corresponding running tapis job before sending notification
         try:
-            valid_state = validate_agave_job(
+            valid_state = validate_tapis_job(
                 job_uuid, job_owner, ['FINISHED', 'FAILED', 'STOPPED']
```

Comment:
This was here before, but wondering why we have `['FINISHED', 'FAILED', 'STOPPED']` being sent to `validate_tapis_job()` as `disallowed_states`?

Reply:
Yeah this was already in the previous code but the reasoning looks to be that the notification should only be sent while the job is still running.